docs/contributing.rst

```diff
@@ -85,3 +85,126 @@ The code style is enforced by `flake8 <https://pypi.org/project/flake8>`_, its
 configuration, and possibly extensions. No code style review should be done by
 a human. All code style enforcement must be automatized to avoid bikeshedding
 and losing time.
+
+
+How-to: Write an Integration
+============================
+
+First off, thank you for writing a new integration! The best way to get started
+writing a new integration is to refer to existing integrations, preferably one
+that instruments a similarly themed library or framework. For example, to write
+a new integration for ``memcached`` we might refer to the existing ``redis``
+integration as a starting point.
+
+The key focus of an integration is to provide concise, insightful information
+about the library or framework that'll be useful for developers to monitor their application as a
+whole. The goal isn't to monitor the library itself per se, but to monitor how
```

Comment:
hmm, this statement isn't entirely true.

it is mostly correct, but sometimes we do have to instrument internal pieces in order to show what is going on. e.g. we can't always just instrument the public api.

Reply:
Also, should try to avoid fuzzy phrases like `per se`, etc.

We can make definitive comments here :)